tools/webpack.config.js

```diff
@@ -129,24 +141,31 @@ const webpackConfig = {
     }),
     new HtmlWebpackHarddiskPlugin({ outputPath: './demo/' }),
     new BannerPlugin({ banner: bannerTemplate, test: /\.js$/ }),
-    new CompressionPlugin({
-      filename: '[path].gz[query]',
-      algorithm: 'gzip',
-      test: /\.(js)$/,
-      threshold: 10240,
-      minRatio: 0.8,
-    }),
+    // new CompressionPlugin({
```

Comment:
does it still generate a compressed version?

Reply:
@kevinchappell no it doesn't, there is a webpack error with this plugin that I'm not sure how to resolve. Also, why is a gzip being output anyway?

Reply:
The compression plugin config needed a [base] added to the config per https://github.com/webpack-contrib/compression-webpack-plugin#string-1 once updated

```
new CompressionPlugin({
      filename: '[path][base].gz[query]',
```